.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "config",
+  "build": {
+    "cacheFrom": "ghcr.io/marinatedconcrete/config-devcontainer:latest",
+    "context": "..",
+    "dockerfile": "Dockerfile"
+  },
+  "customizations": {
+    "vscode": {
+      "extensions": [
+        "esbenp.prettier-vscode",
+        // Enable VSCode to find the prettier package to use when formatting
+        "arcanis.vscode-zipfs",
+        "github.vscode-github-actions",
+        "ms-kubernetes-tools.vscode-kubernetes-tools",
+        "nefrob.vscode-just-syntax",
+        "redhat.vscode-yaml"
+      ],
+      "settings": {
+        "prettier.useEditorConfig": false,
+        "search.exclude": {
+          "**/.yarn": true,
+          "**/.pnp.*": true
+        },
+        "prettier.prettierPath": ".yarn/sdks/prettier/index.js"
+      }
+    }
+  },
+  "features": {
+    "ghcr.io/devcontainers/features/docker-in-docker:2": {
+      // TODO: figure out docker versioning story here
```

Comment:
docs say it's just `version`: https://github.com/devcontainers/features/tree/main/src/docker-in-docker

Reply:
That hasn't yet made it into the debian repo that the install script uses. Gonna see if renovate can learn to look at deb repos, but this approach is feeling increasingly fragile